docs/README.md

```diff
@@ -144,13 +144,15 @@ ecu_info.yaml is the setting file for ECU configuration.
 
 - available_ecu_ids (optional)
 
-  This field specifies a list of all ECU ids, including directly connected, indirectly connected and itself.
+  This field specifies a list of all ECU ids to be updated.  
   Only the main ECU should have this information.  
-  If this field is not specified, value of `ecu_id` is used as this value.  
+  For compatibility, if this `available_ecu_ids` filed does not exist, the ecu_id of the main ecu is considered the entry for available_ecu_ids.
+
+  NOTE: The OTA client user (e.g. Agent) has responsibility for filtering out update requests by using this ecu list.
```

Comment:
What about refining as follow:
```
NOTE: for web.auto user, web.auto agent will use `available_ecu_ids` to generate update request, only ECUs listed in `available_ecu_ids` will be included in the update request list.
```

as otaclient can work not only with web.auto agent, but also other implementation that using otaclient. 
For other implementation, they can use other kinds of implementation to generate the update request. The use of `available_ecu_ids` is required by web.auto agent, but it is not a requirement to other implementation that uses otaclient.

Reply:
@obi-t4 After some consideration, I agree that all OTA client user should follow the spec of `available_ecu_ids`, a.k.a, `available_ecu_ids` should represent the ECUs to be updated.

Reply:
( We can add another line says that `alternative client implementation can also use `available_ecu_ids`, but it is not mandatory.`